llama_stack/distribution/resolver.py

```diff
@@ -11,6 +11,7 @@
 from llama_stack.providers.datatypes import *  # noqa: F403
 from llama_stack.distribution.datatypes import *  # noqa: F403
 
+import llama_stack.distribution.store as distribution_store
```

Comment:
this is not idiomatic in our codebase. can you do 

```
from llama_stack.distribution.store import Registry as DistributionRegistry
```

Reply:
done.